torch/_inductor/codecache.py

```diff
@@ -1534,6 +1534,7 @@ def get_include_and_linking_paths(
         or vec_isa != invalid_vec_isa
         or cuda
         or config.cpp.enable_kernel_profile
+        or config.profiler_mark_wrapper_call
```

Comment:
I'm not sure this is needed here, but I added it based on the assumption that if config.cpp.enable_kernel_profile is part of the conditional, then we should also add it here

Reply:
It is not needed for python wrapper since we use `torch.profiler.record_function` python API. But adding this here should be fine.